source/common/http/utility.cc

```diff
@@ -39,17 +39,20 @@ namespace Utility {
 Http::Status exceptionToStatus(std::function<Http::Status(Buffer::Instance&)> dispatch,
                                Buffer::Instance& data) {
   Http::Status status;
-  try {
+  TRY_NEEDS_AUDIT {
```

Comment:
Is this function actually used? I think it may be dead now that the old codecs are gone.

Reply:
I am not sure. It is still called in codec_impl.cc. I prefer not to do the cleanup for codec in the PR.

Reply:
Again this could should all be removed. I assume this will be done as a follow up?

Reply:
Ping. Are you planning on coming through and doing all of these follow ups?

/wait-any

Reply:
I don't know much about this legacy code, but I will seek help from Yan and Asra to clean them up in the following PRs.